playbooks/robusta_playbooks/pod_enrichments.py

```diff
@@ -33,13 +34,24 @@ def pod_graph_enricher(pod_event: PodEvent, params: PodResourceGraphEnricherPara
     limit_lines = []
     if params.display_limits:
         resource_limits = pod_limits(pod)
+        request_limits = pod_requests(pod)
         if params.resource_type == "CPU" and resource_limits.cpu > 0:
             limit_line = XAxisLine(label="CPU Limit", value=resource_limits.cpu)
-            limit_lines = [limit_line]
+            limit_lines.append(limit_line)
+            if request_limits.cpu > 0:
+                request_memory_limit_in_bytes = request_limits.cpu
+                limit_line = XAxisLine(label="CPU Request", value=request_memory_limit_in_bytes)
```

Comment:
replace 
```
request_memory_limit_in_bytes = request_limits.cpu
limit_line = XAxisLine(label="CPU Request", value=request_memory_limit_in_bytes)
```
with
`limit_line = XAxisLine(label="CPU Request", value=request_limits.cpu)`

Reply:
I had missed the conversion to bytes in the first place. This block should be fine now.